data-lineage/src/main/java/com/wipro/ats/bdre/lineage/entiity/Relation.java

```diff
@@ -152,8 +152,12 @@ public static void updateRelations(List<Table> inTableNodes, List<Table> outTabl
 						Function function = (Function) source;
 						if (function.getOutAlias() != null)
 							outColumnName = function.getOutAlias();
-						else
+						else {
+							if(function.getInNodes().isEmpty()) {
+								//System.out.println("function = " + function);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)